src/renderer.ts

```diff
@@ -351,6 +351,14 @@ const render = (salt: string, pillar: Pillar) => (element: BodyElement, key: num
 
         case ElementKind.Tweet:
             return h(Tweet, { content: element.content, pillar, key });
+
+        case ElementKind.Instagram:
+            const props = {
+                dangerouslySetInnerHTML: {
```

Comment:
Is there anything in the Instagram HTML that prevents us from providing a similar implementation to the one we do [for tweets](https://github.com/guardian/apps-rendering/blob/89b09fde0368f94b2e86a09d16cf2a9aa1999773/src/renderer.ts#L319)?